packages/app-staking/src/Account/index.tsx

```diff
@@ -188,7 +189,8 @@ class Account extends React.PureComponent<Props, State> {
             <AddressMini
               key={index}
               value={nomineeId}
-              withBalance
+              withBalance={false}
```

Comment:
Happy to try, however think the balance is quite useful - 

- you can see (on the contrroller) if you have enough funds to send transactions
- you can see (on the stash) if there is extra funds that you may want to bond

Reply:
Ummm... scratch that, need to look carefully next time - if does show for the accounts actions.